docs/FAQ.md

```diff
@@ -11,33 +11,39 @@ See the [rotation distance document](Rotation_Distance.md).
 
 ## Where's my serial port?
 
-The general way to find a USB serial port is to run `ls
-/dev/serial/by-id/*` from an ssh terminal on the host machine. It will
-likely produce output similar to the following:
+The general way to find a USB serial port is to run
+``` shell
```

Comment:
It seems odd to place a space between the backticks and the designator - all the other places in the docs do not place a space there.

Reply:
I am following the [mkdos-material guide] here
(https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#usage)